plugins/github-pull-requests-board/package.json

```diff
@@ -57,11 +57,11 @@
     "@testing-library/user-event": "^14.0.0",
     "@types/jest": "^26.0.7",
     "@types/node": "^16.11.26",
+    "@types/react": "^16.13.1 || ^17.0.0",
```

Comment:
Is this still required as a regular dependency?

Reply:
I believe so, you can [see in the previous commit](https://github.com/backstage/backstage/pull/13308/commits/9ad1ae83bc10f4b2c2f6db1015c452913b622205) where I moved it but the pipeline check for verifying type dependencies failed.